### PR TITLE
chore(v5-updates): Standardize on React 18, node 18, eslint, and PF v5

### PR DESCRIPTION
**Pull request by dlabaj**
Fixes issue #28 and upgrades to PatternFly version 5.0

### CODE REVIEW

File: packages/module/src/Feedback/FeedbackError.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const FeedbackError = ({ onCloseModal }: FeedbackErrorProps) => {
   const intl = React.useContext(LocaleContext);
   return (
     <div className="chr-c-feedback-success-content">
-      <CheckIcon size="md" color="var(--pf-global--success-color--100)" className="pf-u-mx-auto" />
+      <CheckIcon size={12} color="var(--pf-global--success-color--100)" className="pf-u-mx-auto" />
```
Comment (wise-king-sullyman) [inline packages/module/src/Feedback/FeedbackError.tsx:16]: I think that size has been removed entirely, and that you need to replace this with width and height props (of which I think md corresponds with 1.5em IIRC), but I could be a bit mistaken there.

File: packages/module/src/Feedback/FeedbackSuccess.tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ const FeedbackSuccess = ({ onCloseModal, successTitle, successDescription }: Fee
   const intl = React.useContext(LocaleContext);
   return (
     <div className="chr-c-feedback-success-content">
-      <CheckIcon size="md" color="var(--pf-global--success-color--100)" className="pf-u-mx-auto" />
+      <CheckIcon size={12} color="var(--pf-global--success-color--100)" className="pf-u-mx-auto" />
```
Comment (wise-king-sullyman) [inline packages/module/src/Feedback/FeedbackSuccess.tsx:18]: See previous comment about the size prop on icons.

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
Few comments, also I think you need to update the release.config.js file as well, otherwise it will try to release alpha builds as prerelease versions.

File: packages/module/src/Feedback/FeedbackError.tsx (hunk 1 above)
Comment (dlabaj) [reply packages/module/src/Feedback/FeedbackError.tsx:16]: Okay I think I can just remove it.  It looks like the default size is fine.

File: packages/module/package.json [hunk 3]
```diff
@@ -28,34 +32,31 @@
     "url": "https://github.com/patternfly/react-user-feedback/issues"
   },
   "homepage": "https://github.com/patternfly/react-user-feedback#readme",
-  "publishConfig": {
-    "access": "public"
-  },
```
[the remark this answers is not in the document]
Comment (dlabaj) [reply packages/module/package.json:33]: It's still in the file, I just moved it up.  See lines 7- 10.

File: packages/module/release.config.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  branches: [{ name: 'main', channel: 'prerelease' }],
+  branches: [{ name: 'main', channel: 'alpha' }],
```
Comment (wise-king-sullyman) [inline packages/module/release.config.js:2]: I think if you don't have a prerelease channel semantic-release will throw errors.

Also tagFormat will need to be updated. [Catalog view release config for reference.](https://github.com/patternfly/react-catalog-view/blob/main/packages/module/release.config.js).

You also might want to enable dryRun temporarily to make sure everything works right before we start actually putting releases out.

Comment (dlabaj) [reply packages/module/release.config.js]: @wise-king-sullyman All set thanks.

**Review by wise-king-sullyman — APPROVED** [review]

**Pull request merged**